### DISCUSSION TITLE
固定クエストの報酬のアーティファクトをすでに所持していた場合の報酬

### DISCUSSION THREAD

**Discussion by iks3** [hengband/hengband#237] [Ideas]
固定クエストの報酬がアーティファクトである場合に、すでに所持している場合は、報酬が獲得の巻物になるが、1枚ではクエストによっては価値に見合わないので、調整した方がよいかもしれません。

現状の案としては、固定クエストの難易度(階)を基準にして、獲得、\*獲得\*、アーティファクト生成の巻物あたりから選ぶとどうかという案が出ています。

**Comment by Aquashrimp** [comment]
クローン地獄や古い城なら生成でも良いかもしれない

**Reply by deskull-m** [reply]
報酬が最終級かそれに準ずる★くらいならもうAF生成でいいくらいでしょうな。飛躍しますが、後半の割に合わないクエストも報酬を既存なりそのための新規★に切り替えてそういうの期待させるのもありでしょう。
